File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Soloist/SoloistResultLocationDto.cs
namespace ArenaGestor.APIContracts.Soloist
{
    public class SoloistResultLocationDto
    {
        public int LocationId { get; set; }
        public string Place { get; set; }
        public SoloistResultCountryDto Country { get; set; }
        public string Street { get; set; }
        public int Number { get; set; }
    }
}

File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Soloists/SoloistResultLocationDto.ts
import { SoloistResultCountryDto } from "./SoloistResultCountryDto";

export class SoloistResultLocationDto {
    locationId: Number = 0;
    place: String = "";
    street: String = "";
    number: Number = 0;
    country: SoloistResultCountryDto = new SoloistResultCountryDto();
}